simtools/db_handler.py

```diff
@@ -109,3 +109,38 @@ def collectAllModelParameters(telescopeType, version):
     with open(_yamlFile, 'r') as stream:
         _allPars = yaml.load(stream, Loader=yaml.FullLoader)
     return _allPars
+
+
+def writeModelFile(fileName, destDir):
```

Comment:
Do you plan to actually use this at the moment? With the DB this will be necessary, but I guess now it isn't really no?
Either way, I chose to call files "tabulated data" instead of model files, because the concept of a file is a bit ambiguous. So, if you want to keep this function later, then when I add the DB version it will be called "writeTabulatedData" or something like that, so we can keep both.

Reply:
We can change the name, no problem.
Now now I'm not using it anywhere, I just wrote for you to use as skeleton.
However, whenever I have time I'll start testing the idea of copying the required files to another directory and link sim_telarray to this directory.

Reply:
Another thing, I think we should keep the "Model" in the name somehow.
There might be other kind of data that we will read from databases. Maybe info about previous productions etc. I like the idea of using the label Model for everything related to the Model DB, for the sake of clarity and usability.

Reply:
I would wait with the tests for the DB, no reason to waste your time now with that until it exists. 
At the moment I don't mind keeping "Model" in the name, but I am also a bit hesitant, so I might go back to it in the future.